code/modules/antagonists/brainwashing/brainwashing.dm

```diff
@@ -49,6 +49,11 @@
 		owner_mob.log_message("is no longer brainwashed with the objectives: [english_list(objectives)].", LOG_ATTACK)
 	owner.announce_objectives()
 
+/datum/antagonist/brainwashed/on_mindshield(mob/implanter)
+	owner.remove_antag_datum(/datum/antagonist/brainwashed)
+	if(prob(1) || SSevents.holidays && SSevents.holidays[APRIL_FOOLS])
+		owner.current.say("I'm out! I quit! Whose kidneys are these?", forced = "They're out! They quit! Whose kidneys do they have?")
```

Comment:
This presents a similar class of problem. The message should play for anyone who's de-converted by a mindshield, not just brainwashed people.

So you could do something similar with bitflags. Return a bitflag if an antag datum was lost by the application of the mindshield, and if it was run the chance to play the message